scripts/syncup-s3.sh

```diff
@@ -37,14 +37,14 @@ sync_dirs=(
   scRNA-seq/data/tabula-muris/alevin-quant/10X_P7_12
   scRNA-seq/data/reference
   scRNA-seq/index/Mus_musculus
-  scRNA-seq-advanced/data/ewing-sarcoma/processed
```

Comment:
I don't think you meant to remove this?

Reply:
https://github.com/AlexsLemonade/training-modules/pull/818/commits/6422de75f601000d745d68d7f12d2dc87ded84c1